samza-core/src/main/scala/org/apache/samza/config/JobConfig.scala

```diff
@@ -23,6 +23,7 @@ package org.apache.samza.config
 import java.io.File
 
 import org.apache.samza.container.grouper.stream.GroupByPartitionFactory
+import org.apache.samza.coordinator.metadatastore.KafkaMetadataStoreFactory
```

Comment:
Is this class in samza-core? If not, samza-core shouldn't have a dependency on kafka. If yes, should this be in samza-kafka?